sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_evaluators/_coherence/_coherence.py

```diff
@@ -39,10 +42,16 @@ class CoherenceEvaluator(PromptyEvaluatorBase):
     _RESULT_KEY = "gpt_coherence"
 
     @override
-    def __init__(self, model_config):
+    def __init__(self, model_config, **kwargs):
         current_dir = os.path.dirname(__file__)
         prompty_path = os.path.join(current_dir, self._PROMPTY_FILE)
-        super().__init__(model_config=model_config, prompty_file=prompty_path, result_key=self._RESULT_KEY)
+        passing_score = kwargs.get("passing_score", DEFAULT_PASSING_SCORE)
+        super().__init__(
+            model_config=model_config,
+            prompty_file=prompty_path,
+            result_key=self._RESULT_KEY,
+            passing_score=passing_score,
+        )
 
     @override
     def __call__(
```

Comment:
This can be a follow up item. We should show examples of what the output looks like for each evaluator in the ref doc, and add `x_label` to that with the addtion of `passing_score`.